src/entity/ReputationEvent.ts

```diff
@@ -23,14 +25,16 @@ export enum ReputationType {
   Source = 'source',
 }
 
-const reputationReasonAmount: Record<ReputationReason, number> = {
+export const reputationReasonAmount: Record<ReputationReason, number> = {
```

Comment:
Why are we doing this in separate PR, let's bundle it in the one where it is used? https://github.com/dailydotdev/daily-api/pull/2156#discussion_r1726531910

Reply:
Yes, moving to a separate one because it's needed in mutation too